apps/qubit/modules/digitalobject/actions/editAction.class.php

```diff
@@ -241,6 +298,30 @@ public function processForm()
       $representation->save();
     }
 
+    // Upload new video track files
+    foreach ($this->videoTracks as $usageId => $videoTrack)
+    {
+      if (null !== $uf = $this->form->getValue("trackFile_$usageId"))
```

Comment:
Nitpick: use a more specific variable name, `$uploadedFile` for example.

Reply:
I mostly just copied and modified the code used for uploading other representations.
I'll use `$uploadedTrack` to distinguish a bit. I'll change the variable on line 307 as well.

```suggestion
      if (null !== $uploadedTracks = $this->form->getValue("trackFile_$usageId"))
```